woocommerce/woocommerce
# Check validity before handling submit event in cart

### All Submissions:






### Changes proposed in this Pull Request:

This change will allow HTML5 validation to run when handling keyboard events using `checkValidity()`.

Closes #19996

### How to test the changes in this Pull Request:

1. Add product to cart and visit the cart page
2. Try to input 2.5 quantity. Regular cart update via the button will trigger HTML5 notice.
3. With the field in focus, pressing `return` on your keyboard will submit the form.
4. Apply patch. Now both test cases show the HTML5 validation notice.

### Changelog entry

> Fix cart html5 validation events when using keyboard.
